build/build-image/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:bullseye
+FROM debian:bookworm
```

Comment:
Just looked at the error properly, it's failing in:

```
   Compiling fsevent-sys v4.1.0
   Compiling filetime v0.2.22
Some errors have detailed explanations: E0308, E0412, E0425.
For more information about an error, try `rustc --explain E0308`.
error: could not compile `io-uring` (lib) due to 30 previous errors
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:155: build-macos-binary] Error 101
```

Which is the compilation of the macOS binary. Is there a way to skip using this functionality on macOS? Is it supported on MacOS?